cve_bin_tool/generic_checker.py

```diff
@@ -0,0 +1,175 @@
+from __future__ import annotations
```

Comment:
```suggestion
# Copyright (C) 2023 Intel Corporation
# SPDX-License-Identifier: GPL-3.0-or-later

from __future__ import annotations
```

Needs SPDX license info.  If you don't want to assign copyright to Intel you can put your name in instead.